### PR TITLE
Add a script to check if gitPush1by1 was used

### PR DESCRIPTION
**Pull request by realmarv**
Fixes https://github.com/nblockchain/conventions/issues/82

### CODE REVIEW

File: scripts/detectNotUsingGitPush1by1.fsx [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        )
+        .UnwrapDefault()
+        .Trim()
+        .Split "\n"
+    |> Seq.tail
+
+let notUsingGitPush1by1 =
+    prCommits
+    |> Seq.map(fun commit ->
+        let client: HttpClient = new HttpClient()
```
Comment (knocte) [inline scripts/detectNotUsingGitPush1by1.fsx:41]: @realmarv is client IDisposable? if yes, then it should use `use`, not `let`; if not, then it shouldn't use `new` keyword (this comment applies to other line where you're also using `new`)

**Comment by knocte** [comment]
Two things:
- How is it possible that you didn't need ${{ github.event.pull_request.commits }} ? Are you sure the logic is correct?
- You forgot the "Fixes $URL" at the end.

**Comment by knocte** [comment]
And CI is red actually 😬

**Comment by realmarv** [comment]
> Two things:
> 
> * How is it possible that you didn't need ${{ github.event.pull_request.commits }} ? Are you sure the logic is correct?
> * You forgot the "Fixes $URL" at the end.

`git rev-list HEAD~..HEAD` command returns the PR commits in pull requests. (It returns only the last commit when I try it locally, but in PR, it returns PR commits.)

File: scripts/detectNotUsingGitPush1by1.fsx (hunk 1 above)
Comment (realmarv) [reply scripts/detectNotUsingGitPush1by1.fsx]: Done

File: scripts/detectNotUsingGitPush1by1.fsx [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+let currentBranch =
+    Fsdk
+        .Process
+        .Execute(
+            {
+                Command = "git"
+                Arguments = "rev-parse --abbrev-ref HEAD"
+            },
+            Fsdk.Process.Echo.All
```
Comment (knocte) [inline scripts/detectNotUsingGitPush1by1.fsx:17]: @realmarv let's use Echo.Off

File: scripts/detectNotUsingGitPush1by1.fsx [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+let prCommits =
+    Fsdk
+        .Process
+        .Execute(
+            {
+                Command = "git"
+                Arguments =
+                    sprintf "rev-list %s~..%s" currentBranch currentBranch
+            },
+            Fsdk.Process.Echo.All
```
Comment (knocte) [inline scripts/detectNotUsingGitPush1by1.fsx:31]: @realmarv let's use Echo.Off

**Comment by knocte** [comment]
BTW I pushed 2 WIP commits to test if the build fails, and it does! good work. You can remove those two WIP commits.

File: scripts/detectNotUsingGitPush1by1.fsx (hunk 3 above)
Comment (realmarv) [reply scripts/detectNotUsingGitPush1by1.fsx]: Done

**Pull request merged**